app/validators/user_email_validator.rb

```diff
@@ -23,18 +23,18 @@
   end
 
   def not_allowed_through_configuration?(email)
-    return false if Rails.configuration.x.email_domains_whitelist.blank?
+    return false if Rails.configuration.x.email_domains_allowlist.blank?
 
-    domains = Rails.configuration.x.email_domains_whitelist.gsub('.', '\.')
+    domains = Rails.configuration.x.email_domains_allowlist.gsub('.', '\.')
```

Comment:
👀

Reply:
Not sure whether to ignore or do something about the codeql warning here

Reply:
This is not really a concern, this input is admin-provided and should not contain backslashes in the first place. That being said, I suppose it could make sense to use `Regexp.escape`, but we need to split by `|` first, as this is documented to be a pipe-separated list of domains.

Reply:
> I assume that it previously existed, but as the line is changed, the ignore is invalid and it came up again?

Yes, whatever issue may be here pre-exists this change, and presumably either mattered and was ignored or didnt matter earlier.

If we care, I could attempt to make codeql happy first and then do this rename. If we don't care, probably fine as is.

Reply:
I dismissed the alerts given that:
- this is admin-provided, so if the admin is malicious, well, they're an admin and can do much worse than mess with this regexp
- backslashes are not part of normal input for these fields, so it should not happen by accident either